homeassistant/components/unifiprotect/binary_sensor.py

```diff
@@ -381,6 +400,37 @@ class ProtectBinaryEventEntityDescription(
         ufp_event_obj="last_smart_detect_event",
         ufp_smart_type=SmartDetectObjectType.PACKAGE,
     ),
+    ProtectBinaryEventEntityDescription(
+        key="smart_audio_any",
+        name="Audio Detected",
+        icon="mdi:eye",
+        device_class=DEVICE_CLASS_DETECTION,
```

Comment:
It can be used as a selector for Automations/Blueprints and I do use it the blueprints I made: https://community.home-assistant.io/t/unifi-protect-blueprints/362604

Reply:
Please remove it. That's not part of our design guidelines at the moment.